basic/src/test/java/org/jboss/hal/testsuite/test/runtime/WebMetricsTestCase.java

```diff
@@ -66,7 +67,11 @@ public void before() throws IOException,TimeoutException,InterruptedException {
         statementContext = new DefaultContext();
         ResourceAddress addressStats = ADDRESS_TEMPLATE_STATISTICS.resolve(statementContext);
         dispatcher.execute(new Operation.Builder(ModelDescriptionConstants.WRITE_ATTRIBUTE_OPERATION, addressStats).param("name", "statistics-enabled").param("value", "true").build());
-        cliClient.reload();
+
+        if (cliClient.reloadRequired()) {
+            cliClient.reload();
+            Library.letsSleep(5000);//navigation starts before all servers in domain are reloaded => hangs after clicking on server-one row
```

Comment:
This should be done without need to hardcoding sleeps. To minimize impact for now, I suggest to move the sleep only for case when running the test in domain. 

Note: There is plan to update hal testsuite to use creaper (https://github.com/Creaper-Project/creaper), when there is done its first release to community. Creaper already has support for waiting also for servers in domain to be running and not only the host after reload.